packages/plugin-git/sources/gitUtils.ts

```diff
@@ -0,0 +1,36 @@
+import {execUtils, Configuration}          from '@berry/core';
+import {xfs}                               from '@berry/fslib';
+
+const GitUrlParse = require(`git-url-parse`);
```

Comment:
```suggestion
import GitUrlParse from 'git-url-parse';
```

Reply:
Hmm, tried to change that, but got:

```bash
TS7016: Could not find a declaration file for module 'git-url-parse'.
```

🤔

Reply:
Note that the TS server in VSCode currently must be restarted when you add a new package - maybe it's why you got this error?

Reply:
Also, try moving the `@types/git-url-parse` package into the root manifest (where the others already are)